src/m365/spfx/commands/project/project-upgrade/rules/FN025001_ESLINTRCJS_overrides.ts

```diff
@@ -0,0 +1,432 @@
+import { Finding } from '../../report-model';
+import { Project } from '../../project-model';
+import { TsRule } from './TsRule';
+import ts = require('typescript');
+
+export class FN025001_ESLINTRCJS_overrides extends TsRule {
+  constructor() {
+    super();
+  }
+
+  get id(): string {
+    return 'FN025001';
+  }
+
+  get title(): string {
+    return '.eslintrc.js overrides';
+  }
+
+  get description(): string {
+    return `Add overrides in .eslintrc.js`;
+  }
+
+  get resolution(): string {
+    return `module.exports = {
```

Comment:
We should parametrize this so we can reuse the rule in the future if anything changes again.